loadtest/control/simulcontroller/actions.go

```diff
@@ -433,7 +433,18 @@ func viewChannel(u user.User, channel *model.Channel) control.UserActionResponse
 		return control.UserActionResponse{Err: control.NewUserError(err)}
 	}
 
-	if err := u.GetChannelStats(channel.Id); err != nil {
+	excludeFileCount := true
+	// 1% of the time, users will open RHS, which will include the file count as well.
+	// This is not an entirely accurate representation of events as we are mixing
+	// a normal viewChannel with a viewRHS event
+	// But we cannot distinguish between the two at an API level, so our action
+	// frequencies are also calculated that way.
+	// This is a good enough approximation.
+	if rand.Float64() < 0.01 {
```

Comment:
Does this number come from telemetry, then? Or how did you get it?

Reply:
Nope, just random. Will update from telemetry.

Reply:
It looks like since all RHS events are conflated into one, it's hard to understand how frequently are users opening channel info. I think so far we have to go with 1%. :man_shrugging:

Reply:
It looks sensible, let's go with it.

Reply:
I got the results back and it is around 1.2% approx!! Wow, what an estimate that was!